.github/CODEOWNERS

```diff
@@ -1,3 +0,0 @@
-# AVM core team owns key files
-.github/policies/ @Azure/avm-core-team-technical-terraform
-.github/CODEOWNERS @Azure/avm-core-team-technical-terraform
```

Comment:
Please can we leave this line in? To avoid module owners making changes themselves?